flax/jax_utils.py

```diff
@@ -196,12 +196,22 @@ def scan_in_dim(body_fn, init, xs, axis=(0,), keepdims=False):
     xs: a pytree of tensors to scan over.
     axis: the axis to scan over.
     keepdims: keep the dimensions that are scanned over.
+    unroll: an optional positive integer, or tuple of positive integers
+      showing how many iterations of the loop to be unroll into a single
+      iteration for each axis.
   Returns:
     A tuple of the final carry and the values returned by the body.
   """
   if not isinstance(axis, Iterable):
     axis = (axis,)
 
+  if not isinstance(unroll, Iterable):
+    unroll = (unroll,)
+
+  # Pad unroll with ones so we start unrolling from the innermost loop
```

Comment:
this behavior is missing from the docstring

Reply:
Thanks for the comment, this is now present in the docstring.